_staffers/Bio-CS24-S21-Jared-F.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Jared Flores
+role: Lead ULA
+pronouns: He/Him
+photo: jcf-avatar.png
+---
+
+Hi! My name is Jared. I'm a third year CS major, and I'm really excited as this is my 8th quarter as an Undergraduate Learning Assistant! I've been the Program Lead for the ULA program for the past two years, so whether you're a student looking for some help, or an experienced CS major looking for some opportunities, I'm more than happy to talk to you about any of it.
+
+If you're looking for some industry mentoring too, I worked as a freelance Software Engineer and led the nation's largest high school Cyber Security program, as such I'm well-versed in Windows, Linux, and CISCO networking. In my free time, you'll often find my in CSIL or the UCSB Library working late into the nights on CS labs or working on games.
+
+I love game development and entertainment based applications, so if you ever have the inspiration to create the next big MMO or even a small singleplayer platformer, definitely reach out on Piazza -- I'm absolutely ready to help mentor or be a team member. Coding is an art to me; I'm in love with the intricacies and logical nuances that it entails.
```

Comment:
@jay-red Since we don't use Piazza, do you want to provide an alternative way for them to reach you without listing your direct contact info? :-)